migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,62 @@
+[id='Migration-toolkit-for-containers-overview']
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: Migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the MigrationToolkit for Containers (MTC). To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[MTC installation]: You can install the Migration Toolkit for Containers (MTC) on {product-title} 4. After installing MTC, you must configure an object storage to use as a replication repository. You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You can do the following as a part of installing MTC:
```

Comment:
The wording "You can do the following" doesn't match up with the following list, which aren't actions. Suggest changing the list to starting with verbs.

Also, per the IBM style guide, we shouldn't use "following" alone, but should say something after it, like "the following tasks" or similar.

Reply:
acked